### PR TITLE
Bugfix: fixes issue with TS upgrade introducing issue with adding events to window/svg

### PR DESCRIPTION
**Pull request by scalvert**
Addresses #162

### CODE REVIEW

**Comment by scalvert** [comment]
cc: @pgrippi

File: addon/mixins/dom.ts [hunk 1]
```diff
@@ -144,12 +144,11 @@ function findElement(
 ): HTMLElement {
   let selectorType: string = typeof selector;
   let element;
-
+debugger;
```
Comment (rwjblue) [inline addon/mixins/dom.ts]: 🔪

Comment (scalvert) [reply addon/mixins/dom.ts]: oh good lord...

**Comment by scalvert** [comment]
They were. Both were failing.

**Pull request merged**